### PR TITLE
Adds constant_tags to ThreadStats

### PR DESCRIPTION
**Pull request by jofusa**
Added the same constant_tags to the ThreadStats class as I had done to the blocking version.

### CODE REVIEW

**Comment by yannmh** [comment]
Looking great ! Again, thanks @jofusa !

We'll merge your PR shortly so your changes can be shipped with the next release :ship:

File: datadog/threadstats/base.py [hunk 1]
```diff
@@ -116,6 +122,13 @@ def event(self, title, text, alert_type=None, aggregation_key=None,
             'The web server is up again', alert_type='success')
         """
         if not self._disabled:
+            # Append all client level tags to every event
+            if self.constant_tags:
+                if tags:
+                    tags += self.constant_tags
+                else:
+                    tags = self.constant_tags
+
```
Comment (yannmh) [inline datadog/threadstats/base.py]: `constant_tags` are not associated to events in `dogstatsd`. To be consistent, we should do the same with `ThreadStats`, what do you think ?

Comment (jofusa) [inline datadog/threadstats/base.py]: Or I can add to dogstatsd. We have a use case for it.

**Comment by jofusa** [comment]
Added constant tags to base agent. let me know if you like it, or i can rip it out

**Comment by yannmh** [comment]
Thanks a lot @jofusa :exclamation: 
Changes will be released later today in a v0.10.0 version :ship:

**Pull request merged**